src/service/vsync-impl.js

```diff
@@ -365,6 +385,20 @@ export class Vsync {
    */
   runScheduledTasks_() {
     this.scheduled_ = false;
+    if (this.jankRateDisplay_) {
+      const paintLatency =
+          Math.floor(this.win.performance.now() - this.scheduledTime_);
+      if (paintLatency > FRAME_TIME) {
+        this.jankCounter_++;
+        if (paintLatency > 100) {
+          this.bigJankCounter_++;
+        }
+        this.jankRateDisplay_.textContent =
```

Comment:
That is a bit of a misunderstanding. Reading it never layouts. That is the big (and unintuitive difference from `innerText`). Writing it can obviously cause layout, since changing text can change layout.